libraries/client/src/apis.js

```diff
@@ -180,6 +180,22 @@ module.exports = {
           "title": "List Roles",
           "type": "function"
         },
+        {
+          "args": [
+          ],
+          "description": "Get a list of all roles, each role object also includes the list of\nscopes it expands to. Since this\nlist may become long, callers can use the `limit` and `continuationToken`\nquery arguments to page through the responses.",
+          "method": "get",
+          "name": "listRoles2",
+          "output": "v1/list-roles2-response.json#",
+          "query": [
+            "continuationToken",
+            "limit"
+          ],
+          "route": "/roles2/",
+          "stability": "stable",
+          "title": "List Roles",
+          "type": "function"
+        },
```

Comment:
Let's remove the change to this file.  It will get regenerated eventually, but at the moment it differs from the API definition (in `title` at least)